src/modules/commander/commander_params.c

```diff
@@ -953,3 +953,17 @@ PARAM_DEFINE_INT32(COM_FLT_PROFILE, 0);
  * @boolean
  */
 PARAM_DEFINE_INT32(COM_ARM_CHK_ESCS, 1);
+
+/**
+ * Condition to enter prearmed mode
+ *
+ * Condition to enter the prearmed state, an intermediate state between disarmed and armed
+ * in which non-throttling actuators are active.
```

Comment:
What is a non-throttling activator?

Reply:
A non throttling actuator is an actuator that does not drive a motor/engine. The underlying assumption is that throttling actuator is dangerous and should not move until the system is armed, while non-throttling actuators (ailerons, flaps, camera gimbal, etc) are safer and can move before the system is armed.

Right now, throttling actuator is defined here (NaN means the actuator will stay in disarmed position) https://github.com/PX4/Firmware/blob/a20b508d7e60a1332e3c9ff18d69df4d5e04eb5d/src/drivers/px4fmu/fmu.cpp#L1062-L1070. But I would not bother documenting that too precisely as this is likely to change when we introduce metadata at the mixer level to indicate which output is safe to prearm.